quickwit-metastore/migrations/2021-08-29-122131_create_splits/up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE splits (
+    split_id VARCHAR(50) PRIMARY KEY, 
+    split_state INTEGER NOT NULL, 
```

Comment:
Let's use strings instead of INTEGER for the split state.

PostgreSQL also has enum that supposedly offer the best of two worlds, but they are a pain to modify.

Reply:
I changed to use VARCHAR instead of INTEGER for split_state.